commit 0c227640b34a1231a41d190a4c247de1b4347c3e
Author: Anatol Pomozov <anatol.pomozov@gmail.com>
Date:   Sat Feb 16 12:56:33 2019 -0800

    Fix compilation for Arch's libmagic

diff --git a/ext/RMagick/extconf.rb b/ext/RMagick/extconf.rb
index 40b0a535..864eb173 100644
--- a/ext/RMagick/extconf.rb
+++ b/ext/RMagick/extconf.rb
@@ -66,7 +66,7 @@ module RMagick
 
         magick_package = determine_imagemagick_package
 
-        $magick_version = `pkg-config #{magick_package} --modversion`[/^(\d+\.\d+\.\d+)/]
+        $magick_version = `PKG_CONFIG_PATH=/usr/lib/imagemagick6/pkgconfig pkg-config #{magick_package} --modversion`[/^(\d+\.\d+\.\d+)/]
 
         check_multiple_imagemagick_versions
         check_partial_imagemagick_versions
@@ -80,10 +80,10 @@ module RMagick
         end
 
         # Save flags
-        $CFLAGS = ENV['CFLAGS'].to_s + ' ' + `pkg-config --cflags #{magick_package}`.chomp
-        $CPPFLAGS = ENV['CPPFLAGS'].to_s + ' ' + `pkg-config --cflags #{magick_package}`.chomp
-        $LDFLAGS = ENV['LDFLAGS'].to_s + ' ' + `pkg-config --libs #{magick_package}`.chomp
-        $LOCAL_LIBS = ENV['LIBS'].to_s + ' ' + `pkg-config --libs #{magick_package}`.chomp
+        $CFLAGS = ENV['CFLAGS'].to_s + ' ' + `PKG_CONFIG_PATH=/usr/lib/imagemagick6/pkgconfig pkg-config --cflags #{magick_package}`.chomp
+        $CPPFLAGS = ENV['CPPFLAGS'].to_s + ' ' + `PKG_CONFIG_PATH=/usr/lib/imagemagick6/pkgconfig pkg-config --cflags #{magick_package}`.chomp
+        $LDFLAGS = ENV['LDFLAGS'].to_s + ' ' + `PKG_CONFIG_PATH=/usr/lib/imagemagick6/pkgconfig pkg-config --libs #{magick_package}`.chomp
+        $LOCAL_LIBS = ENV['LIBS'].to_s + ' ' + `PKG_CONFIG_PATH=/usr/lib/imagemagick6/pkgconfig pkg-config --libs #{magick_package}`.chomp
 
         set_archflags_for_osx if RUBY_PLATFORM =~ /darwin/ # osx
 
@@ -162,11 +162,11 @@ SRC
         exit_failure "Can't install RMagick #{RMAGICK_VERS}. Can't find pkg-config in #{ENV['PATH']}\n"
       end
 
-      packages = `pkg-config --list-all`.scan(/(ImageMagick\-6[\.A-Z0-9]+) .*/).flatten
+      packages = `PKG_CONFIG_PATH=/usr/lib/imagemagick6/pkgconfig pkg-config --list-all`.scan(/(ImageMagick\-6[\.A-Z0-9]+) .*/).flatten
 
       # For ancient version of ImageMagick 6 we need a different regex
       if packages.empty?
-        packages = `pkg-config --list-all`.scan(/(ImageMagick) .*/).flatten
+        packages = `PKG_CONFIG_PATH=/usr/lib/imagemagick6/pkgconfig pkg-config --list-all`.scan(/(ImageMagick) .*/).flatten
       end
 
       if packages.empty?
@@ -295,7 +295,7 @@ END_MINGW
     def assert_has_dev_libs!
       return unless RUBY_PLATFORM !~ /mswin|mingw/
 
-      unless `pkg-config --libs MagickCore`[/\bl\s*(MagickCore|Magick)6?\b/]
+      unless `PKG_CONFIG_PATH=/usr/lib/imagemagick6/pkgconfig pkg-config --libs MagickCore`[/\bl\s*(MagickCore|Magick)6?\b/]
         exit_failure "Can't install RMagick #{RMAGICK_VERS}. " \
                    "Can't find the ImageMagick library or one of the dependent libraries. " \
                    "Check the mkmf.log file for more detailed information.\n"
